<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormLogin</class>
 <widget class="QWidget" name="FormLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="widgetTop" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>512</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:600; color:#ffffff;&quot;&gt;Login&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>512</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLineEdit" name="passwordEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;黑体&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="userEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;用户&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;密码&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QToolButton" name="loginButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="remberPassCheckBox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>110</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>记住密码</string>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="autoLoginCheckBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>自动登录</string>
    </property>
   </widget>
   <widget class="QToolButton" name="registToolButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>注册账号</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>userEdit</tabstop>
  <tabstop>passwordEdit</tabstop>
  <tabstop>loginButton</tabstop>
  <tabstop>remberPassCheckBox</tabstop>
  <tabstop>autoLoginCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
